server/core/src/initServer.cpp

```diff
@@ -683,44 +675,9 @@ cleanupAndExit( int status ) {
 
 void
 signalExit( int ) {
-#if 0
-    // RTS - rodsLog calls in signal handlers are unsafe - #3326
-    rodsLog( LOG_NOTICE,
-             "caught a signal and exiting\n" );
-#endif
     cleanupAndExit( SYS_CAUGHT_SIGNAL );
 }
 
-void
-rsPipeSignalHandler( int ) {
```

Comment:
nothing uses this so i removed it.

any objections?

Reply:
did you get the one at
```
./server/core/include/initServer.hpp:36:rsPipeSignalHandler( int );
```

Reply:
good catch.

removed.